gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/GenerateDataTemplateTask.java

```diff
@@ -127,8 +127,20 @@ public void setEnableArgFile(boolean enable)
     _enableArgFile = enable;
   }
 
-  @Input
+  /**
+   * @deprecated by {@link #isGenerateFieldMask()} because Gradle 7 requires
+   *     input and output properties to be annotated on getters, which have a
+   *     prefix of "is" or "get".
+   */
+  @Deprecated
   public boolean generateFieldMask()
+  {
+    return isGenerateFieldMask();
+  }
+
+  @Optional
```

Comment:
The old accessor was _not_ optional; is this OK?

Reply:
Why do we need a `Boolean` wrapper? The old accessor was a primitive.

Reply:
Good call, I changed it back to a primitive and made it default to `false`.